pkg/glusterfs/deploy/kubernetes/csi-attacher-glusterfsplugin.yaml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
I don't understand how the project is being licensed. We have LGPLv3 and GPLv2 COPYING files added, but these K8S definition files are licensed as Apache2!? I have 2 questions.

1. Why?
2. How is it supposed to work?

Reply:
@kshlm as mentioned in my last comments , license part is unclear and we need to reach to conclusion. some discussions were on https://github.com/gluster/gluster-csi-driver/pull/1